app/views/admin/billboards/index.html.erb

```diff
@@ -31,23 +31,23 @@
       </tr>
     </thead>
     <tbody class="crayons-card">
-      <% @billboards.each do |display_ad| %>
-          <tr data-row-id="<%= display_ad.id %>">
-            <td><%= link_to display_ad.name, edit_admin_billboard_path(display_ad) %></td>
-            <td><%= display_ad.human_readable_placement_area %></td>
-            <td><%= display_ad.display_to %></td>
-            <td><%= display_ad.type_of.titleize %></td>
-            <% if display_ad.published? && display_ad.approved? %>
+      <% @billboards.each do |billboard| %>
+          <tr data-row-id="<%= billboard.id %>">
```

Comment:
Not related to the main purpose, but started renaming variabes in this file and decided to keep.